client/src/pages/chatlink/components/button.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import '../src/pages/chatlink/style.css';
+
+const Button = (props) => {
```

Comment:
Let's destructure the props here, will be more readable and look clean.
`const { type, state, onClick, width } = props`